packages/amplify_api/ios/Classes/FlutterApiErrorMessage.swift

```diff
@@ -19,6 +19,8 @@ enum FlutterApiErrorMessage: String {
     case MALFORMED = "AMPLIFY_REQUEST_MALFORMED"
     case QUERY_FAILED = "AMPLIFY_API_QUERY_FAILED"
     case MUTATE_FAILED = "AMPLIFY_API_MUTATE_FAILED"
+    case SUBSCRIBE_FAILED = "AMPLIFY_API_SUBSCRIBE_FAILED"
+    case ESTABLISH_SUBSCRIPTION_FAILED = "AMPLIFY_API_FAILED_TO_ESTABLISH_SUBSCRIPTION"
```

Comment:
This change shouldn't be necessary, right?  If anything, we would be casting to a new exception type?

Reply:
This change was based on the original error handling approach. I've now updated the code based on the error handling refactor.